javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var clickCounter = 0;
+var $lightBulb = document.querySelector('.light-bulb');
+var $container = document.querySelector('.container');
+
+function lightOnAndOff(event) {
+  clickCounter++;
+  if (clickCounter % 2 !== 0) {
+    $lightBulb.className = 'buttonClick';
+    $container.className = 'lightOn';
+  } else if (clickCounter % 2 === 0) {
```

Comment:
This can just be an `else` statement.

Reply:
![Screen Shot 2021-02-02 at 12 53 29 PM](https://user-images.githubusercontent.com/74885918/106661386-00cd9d00-6556-11eb-83bd-db65ff2253da.png)
![Screen Shot 2021-02-02 at 12 53 14 PM](https://user-images.githubusercontent.com/74885918/106661408-062ae780-6556-11eb-9d27-f9460a921e40.png)